woocommerce/woocommerce-ios
# Store creation profiler: Update wording and assets for loading screen

Closes: #10421 


## Description

This PR updates the UI for the loading screen of the store creation flow. The number of steps for the loading is reduced and new wording and assets have been updated. Similar to the update of the profiler questions, this update is applied immediately without the check for the feature flag of the updated store creation flow for simplicity.

## Testing instructions

- Log in to the app and switch to the Menu tab.
- Select Add a store > Create a new store.
- Complete the profiler questions and tap Start free trial. The loading screen should then be displayed with the updated UI following the design 8sNT07kW3VEdnor4baI6ft-fi-387_40366.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/6288906a-7246-408b-ad86-5e281f333957



---
